contracts/polygon/connectors/aave/automation-v3/main.sol

```diff
@@ -0,0 +1,69 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @title InstaAutomation
+ * @dev Insta-Aave-v3-Automation
+ */
+
+import "./events.sol";
+import "./interfaces.sol";
+
+abstract contract Resolver is Events {
+	InstaAaveAutomation internal immutable automation =
+		InstaAaveAutomation(0x7eE533CB0642f18191D2927bdA735c3830B355BB);
+
+	function submitAutomationRequest(
+		uint256 safeHealthFactor,
+		uint256 thresholdHealthFactor
+	)
+		external
+		payable
+		returns (string memory _eventName, bytes memory _eventParam)
+	{
+		automation.submitAutomationRequest(
+			safeHealthFactor,
+			thresholdHealthFactor
+		);
+
+		(_eventName, _eventParam) = (
+			"LogSubmitAutomation(uint256,uint256)",
+			abi.encode(safeHealthFactor, thresholdHealthFactor)
+		);
+	}
+
+	function cancelAutomationRequest()
+		external
+		payable
+		returns (string memory _eventName, bytes memory _eventParam)
+	{
+		automation.cancelAutomationRequest();
```

Comment:
@pradyuman-verma we need to remove automation contract as auth

Reply:
Okay